atlassian_jwt_auth/contrib/tests/django/test_django.py

```diff
@@ -0,0 +1,155 @@
+import os
+
+import django
+from django.test.testcases import SimpleTestCase
+from django.test.utils import override_settings
+try:
+    from django.urls import reverse
+except ImportError:
+    from django.core.urlresolvers import reverse
+
+from atlassian_jwt_auth import create_signer
+from atlassian_jwt_auth.contrib.tests.utils import get_static_retriever_class
+from atlassian_jwt_auth.tests import utils
+from atlassian_jwt_auth.tests.utils import RS256KeyTestMixin
+
+
+def create_token(issuer, audience, key_id, private_key):
+    signer = create_signer(issuer, key_id, private_key)
+    return signer.generate_jwt(audience)
+
+
+class TestAsapDecorator(RS256KeyTestMixin, SimpleTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        os.environ.setdefault(
+            'DJANGO_SETTINGS_MODULE',
+            'atlassian_jwt_auth.contrib.tests.django.settings')
+        django.setup()
```

Comment:
Do you really have to do django.setup() if you're running inside a Django test case?